runtime/doc/diagnostic.txt

```diff
@@ -180,6 +180,30 @@ the `virtual_lines` handler with the following keymap: >lua
     end, { desc = 'Toggle diagnostic virtual_lines' })
 <
 
+                                            *diagnostic-on-jump-example*
+You can use the `on_jump` option from |vim.diagnostic.jump()| to show the
+diagnostic that was jumped to using a specific handler. For example, the
+following uses the `virtual_lines` when jumping to a diagnostic: >lua
+
+  local virt_lines_ns = vim.api.nvim_create_namespace 'on_diagnostic_jump'
```

Comment:
I tested this with my own configuration and the separate namespace is needed to avoid changing the other diagnostics from the same namespace.